L/libint/build_tarballs.jl

```diff
@@ -0,0 +1,60 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libint"
+version = v"2.7.0-beta.5"
```

Comment:
We want to use the same version as the upstream package, as long as this makes Pkg happy.  Unfortunately the "prerelease" string can cause some problems (for example for sorting), so better to just drop this
```suggestion
version = v"2.7.0"
```